src/utils/classifier_report.py

```diff
@@ -0,0 +1,125 @@
+# this command-line tool can take two arguments which must be preceded by their respective flags
+#   1) -f : (REQUIRED) The CSV file that has four columns: 
+#                      original_path, normalized_path, predicted_class, predicted_conf
+#   2) -s : (OPTIONAL) The path to the saved html path. If this argument is not specified, 
+#                      a 'reports' directory will be created where 'reports.html' will be saved.
+
+
+from airium import Airium
+from pathlib import Path
+from csv import DictReader
+import webbrowser
+import argparse
+import os
+
+
+class ReportGenerator:
+    def __init__(self):
+        self.parse_args()
```

Comment:
I would suggest only calling parse_args and parse_csv in the initializer, since they are recording inputs and initializing the state of the report generator. Creating the html report, saving it, and launching it in a web browser seem unexpected to me for an initializer and would make reuse and testing harder. So I would suggest calling those other actions as additional steps.